config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
+- [ ] PR Title has a short summary of what changes are included
+- [ ] PR Description below includes additional detail that isn't included in the title summary
+- [ ] Assign applicable leads to your PR for review
```

Comment:
Are we only going to have leads review PRs?   It seems to me that someone else on the project could review a PR especially if the lead is not integrated into the project.  

The lead may still need to do a quick review but I don't know that leads should be the only ones reviewing code. 

Also, who reviews the code for leads?  This doesn't seem applicable in all situations.